libs/remixd/src/bin/remixd.ts

```diff
@@ -63,6 +63,7 @@ function errorHandler (error: any, service: string) {
     .description('Establish a two-way websocket connection between the local computer and Remix IDE for a folder')
     .option('-u, --remix-ide  <url>', 'URL of remix instance allowed to connect')
     .option('-s, --shared-folder <path>', 'Folder to share with Remix IDE (Default: CWD)')
+    .option('-i, --install <name>', 'Module name to install locally')
```

Comment:
coud you this be run be default in a post install script?

Reply:
this is done now.